src/pudl_archiver/orchestrator.py

```diff
@@ -34,6 +41,22 @@ class Config:
         arbitrary_types_allowed = True
 
 
+class FileWrapper(io.BytesIO):
```

Comment:
When `aiohttp` times out while uploading a file, it will then close the file, which leads to an error when we retry the upload. I created this wrapper so `aiohttp` can't close the file, which doesn't seem like the best solution, but it was easy, and maybe it's good enough?

Reply:
The discussion on [this issue](https://github.com/aio-libs/aiohttp/issues/1907) implies that there's some sort of custom Payload we could pass in, and that this is the preferred method by the aiohttp maintainer. But, it's been a whole major version since then, and the documentation doesn't mention this at all.

[This issue](https://github.com/aio-libs/aiohttp/issues/3654) uses a "monkeypatch the `.close()` function" approach which seems equally janky but a little shorter. 🤷 

This existing option is good enough for me, though we should add some more documentation about this somewhat surprising behavior.

Reply:
Yeah I found an issue where the `aiohttp` maintainers seemed quite convinced that they should take ownership of a file, and callers should reopen the file, which would requrie a lot of re-engineering on our part, so gonna have to live with a hack I guess.